calcom/cal.com
# Fix/force username after signup

## What does this PR do?



Fixes #2018
When you create a new account on Cal.com and you do not have username, it going to give you one with your name and some random letters. I only can tested create accounts from Prisma Data Platform. 
![image](https://user-images.githubusercontent.com/19562383/163957290-6d7e2d05-949b-42d4-89ac-7c5ab27f0b8a.png)

Once the registration is complete you can change the username in settings.
## Type of change




## How should this be tested?



1. Add a new user to the team.
2. Sign up with the new user. (This user has to be created in DB). 
3. Once the registration is complete you can see a random username.
4. Go back to the owner team account and open view public page, booking and select any event
![image](https://user-images.githubusercontent.com/19562383/163958668-edd8ad1f-d653-4f69-870d-439d495cf5c9.png)
